### ISSUE TITLE
Alpaca not compatible w/ jquery.3.4.0

### ISSUE THREAD

**Issue by seanvree** [gitana/alpaca#705]
I have been using Alpaca with jquery-3.3.1.min, but recently updated to jquery-3.4.0.min When the form loads I get these errors:

```
jquery.min.js:2 Uncaught RangeError: Maximum call stack size exceeded
    at String.replace (<anonymous>)
    at V (jquery.min.js:2)
    at Y.set (jquery.min.js:2)
    at De (jquery.min.js:2)
    at Text.setup (jquery.min.js:2)
    at Object.add (jquery.min.js:2)
    at De (jquery.min.js:2)
    at Text.setup (jquery.min.js:2)
    at Object.add (jquery.min.js:2)
    at De (jquery.min.js:2)
```

![image](https://user-images.githubusercontent.com/8906637/56497541-2f7e9e80-64bb-11e9-9065-67050b6d4f33.png)

Any ideas?

**Comment by ambischof** [comment]
I don't know the solution but I was able to isolate the problem. When it tries to focus the control, the `control` prop is a jquery selector that has `text` nodes in it, which jquery 3.4 doesn't handle well.

You can recreate this by doing the following: 
```
$($('<div>text<span>span</span></div>').get(0).firstChild).focus(function() {})
```
Here it's isolating the text node, wrapping it in query, and calling focus on it, resulting in that same error.

**Comment by raviatot** [comment]
in alpacajs, from the below radio templates remove un-neccessary new lines and blank spaces would resolve this issue.

**this["HandlebarsPrecompiled"]["web-edit"]["control-radio"] = Handlebars.template({"1":function(depth0,helpers,partials,data) {**

**Comment by seanvree** [comment]
So has this been resolved?

**Comment by seanvree** [comment]
> in alpacajs, from the below radio templates remove un-neccessary new lines and blank spaces would resolve this issue.
> 
> **this["HandlebarsPrecompiled"]["web-edit"]["control-radio"] = Handlebars.template({"1":function(depth0,helpers,partials,data) {**

I'm not sure what you mean @raviatot .  Are you saying in the alpaca.min.js file or a template that I'm using?

**Comment by jabrah** [comment]
Not sure if this is still being tracked, but it seems like this issue was resolved with jQuery 3.5.0+

**Comment by shenyu-li** [comment]
no, I am still having this issue with jQuery 3.5.0 and 3.5.1 'Maximum call stack size exceeded', any fix on this?

**Comment by jasondalycan** [comment]
Same issue.  Is there an update?
